__tests__/TreeNode.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import TreeNode from '../src/components/TreeNode';
+import defaultTheme from '../src/themes/default';
+import defaultAnimations from '../src/themes/animations';
+import defaultDecorators from '../src/components/Decorators';
+import data from '../example/data';
+
+describe('<TreeNode/>', () => {
+    describe('when onToggle function is called', () => {
+        test('should call correctly', () => {
+            const clickFn = jest.fn();
+            const wrapper = shallow(<TreeNode
```

Comment:
you can create a function on this test file called `renderComponent` to render with default props and overwrite when you need.
```javascript
const renderComponent = (props = {}) => <TreeNode
  {...{
    decorators={defaultDecorators}
    ...props,
  }}
/>;
```
usage:
```javascript
const wrapper = shallow(renderComponent());
...
const wrapper = shallow(renderComponent({node: customNode}));
```